bot/handlers/user/subscription/core.py

```
@@ -125,28 +198,53 @@ async def my_subscription_command_handler(
+        return str(val)
+
+    if traffic_mode:
+        limit_display = _fmt_gb(active.get("traffic_limit_bytes"))
+        used_display = _fmt_gb(active.get("traffic_used_bytes"))
+        remaining_display = get_text("traffic_na")
+        try:
+            limit_val = active.get("traffic_limit_bytes") or 0
+            used_val = active.get("traffic_used_bytes") or 0
+            remaining_val = max(0, float(limit_val) - float(used_val))
+            remaining_display = _fmt_gb(remaining_val)
+        except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)